source/appModules/securecrt.py

```diff
@@ -1,23 +1,28 @@
 #appModules/securecrt.py
+# Modified to include a regular expression so that all Secure CRT versions are supported past, present, and future.
+# Modification of original securecrt.py by Noel Romey 9/2016
 #A part of NonVisual Desktop Access (NVDA)
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
-#Copyright (C) 2010-2012 NV Access Limited
+#Copyright (C) 2010-2016 NV Access Limited
 
-"""App module for SecureCRT
+"""App module for automatic reading of terminal windows in all versions of SecureCRT past, present and future
 """
 
 import oleacc
+import re
 from NVDAObjects.behaviors import Terminal
 from NVDAObjects.window import DisplayModelEditableText, DisplayModelLiveText
 import appModuleHandler
-
+#initialize a compiles regexp object for all window names following the standard SecureCrt Window naming comvention for all versions
+MATCHER = re.compile ("AfxFrameOrView\d{2,}u")

```

Comment:
Perhaps rename MATCHER to something like RE_TERMINAL_WCLASS to make it clear what it's for. The reason is that this app module might also serve some additional purpose in future, at which point the name MATCHER might be confusing. Sorry; I should have picked this up in my previous review.